csharp-basic-topics/RemoveIllegalCharsFromPathAndFilenames/RemoveIllegalCharsFromPathAndFilenames/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Text.RegularExpressions;
+
+namespace CodeMazeGuides.CSBasic
```

Comment:
Namespace name should be the CSPROJ name.

Reply:
You can use here the top level statements approach, so instead of defining the Program class you just start writing the content of the `Main()` method block